config/api_test.go

```diff
@@ -15,16 +15,11 @@ func assertViperDefault(t *testing.T, key string, expected string) {
 
 func TestAPIDefault(t *testing.T) {
```

Comment:
test all config constants in config/api.go

Reply:
@NicolasMahe is working on a refactoring of the configs, this will be done on his PR. I don't want to touch much the configs to avoid a lot of conflicts